### ISSUE TITLE
where clause URL appears to yield a 404

### ISSUE THREAD

**Issue by wasade** [qiime2/q2-feature-table#61]
These [lines](https://github.com/qiime2/q2-feature-table/blob/master/q2_feature_table/plugin_setup.py#L112-L115) specifically. Tested using 2.0.6, which shows the following URL:

https://docs.qiime2.org/2.0.6/tutorials/table-filtering.html

**Comment by thermokarst** [comment]
Hi @wasade, the tutorial (https://github.com/qiime2/docs/pull/39) is slated for the February release cycle. Thanks!

**Comment by wasade** [comment]
So https://docs.qiime2.org/2.0.6/tutorials/table-filtering/ shouldn't be live?

**Comment by wasade** [comment]
For a little more context, the URL listed in the help text below is invalid

```bash
$ qiime feature-table filter-features --help
Usage: qiime feature-table filter-features [OPTIONS]

  Filter features from table based on frequency and/or metadata. Any samples
  with a frequency of zero after feature filtering will also be removed. If
  no value(s) are provided for max_frequency and/or max_samples, they will
  default to infinity (i.e., no maximum frequency and/or sample filter will
  be applied).

  The where parameter takes a SQLite WHERE clause. See the table filtering
  tutorial for additional detail:
  https://docs.qiime2.org/2.0.6/tutorials/table-filtering.html

Options:
  --i-table PATH                  Artifact: FeatureTable[Frequency]
                                  [required]
  --m-feature-metadata-file PATH  Metadata mapping file  [optional]
  --p-max-frequency INTEGER       [optional]
  --p-max-samples INTEGER         [optional]
  --p-min-frequency INTEGER       [default: 0]
  --p-min-samples INTEGER         [default: 0]
  --p-where TEXT                  [optional]
  --o-filtered-table PATH         Artifact: FeatureTable[Frequency]  [required
                                  if not passing --output-dir]
  --output-dir DIRECTORY          Output unspecified results to a directory
  --cmd-config PATH               Use config file for command options
  --verbose                       Display verbose output to stdout and/or
                                  stderr during execution of this action.
                                  [default: False]
  --help                          Show this message and exit.
```

**Comment by thermokarst** [comment]
Right, the tutorial is up there now (https://docs.qiime2.org/2.0.6/tutorials/table-filtering/), but the URL listed in the help text has an extra `.html` at the end. I was confused about what you were asking about initially. The fix will be to just drop the suffix in the help text URL. Also, I think this happened because the URL was specified before we actually had a tutorial in place, so this was a bit of a placeholder. Thanks!

**Comment by thermokarst** [comment]
Also, I totally spaced that we decided to update the `2.0.6` docs to include this tutorial. This makes sense though, because the functionality is present in the `2.0.6` release.

**Comment by wasade** [comment]
Thanks!

On Jan 6, 2017 07:29, "Greg Caporaso" <notifications@github.com> wrote:

Closed #61 <https://github.com/qiime2/q2-feature-table/issues/61> via #62
<https://github.com/qiime2/q2-feature-table/pull/62>.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/qiime2/q2-feature-table/issues/61#event-914164196>, or mute
the thread
